Standalone/Dockerfile.txt

```diff
@@ -1,5 +1,7 @@
 USER 1200
 
+RUN sudo chmod +x /dev/shm
```

Comment:
This should be done in the `BaseNode` image

Reply:
Okay, I've updated it with the change in NodeBase image and I tested it there and it works if you want to look at this PR updated.

Reply:
@diemol It's now in NodeBase Dockerfile, I'm not in love with my comment I made....I was trying to follow your Dockerfile "comments pattern" you guys were doing, feel free to change as you'd like.

Reply:
I see it in `Base`, not in `NodeBase`, am I missing something?

Reply:
Sorry, I didn't pay close enough attention and I did put in in Base, putting it in NodeBase now, then I'll test it again and update the PR.

Reply:
@diemol Fix is now in NodeBase, I tested that it works. Feel free to change my code comment in NodeBase.

Reply:
@diemol anything else needed? I think this is all good now?

Reply:
@theddub before merging this, did you check if the image size increased? If so, how much?

Reply:
@diemol I just checked, base(obviously not effected now), node-base, node-chrome & standalone-chrome all stayed exactly the same as reported( to the 1/100 of a MB as seen in Docker Desktop...ie...doesn't round to the nearest MB like "docker image ls" does) by Dokcer(docker image ls) and Dokcer Desktop(GUI). No change whatsoever before and after.